README.md

```diff
@@ -1,58 +1,173 @@
+- support for [non-Markdown input formats](https://zola.discourse.group/t/alternative-input-formats/76)
+- support for [non-HTML output formats](https://zola.discourse.group/t/proposal-custom-output-formats/68/4)
 
-Hugo gets ![ehh] for the template engine because while it is probably the most powerful template engine in the list (after Jinja2) it personally drives me insane, to the point of writing my own template engine and static site generator. Yes, this is a bit biased.
+### Translations
 
-Zola gets ![ehh] for multi-language support as it only has a basic support and does not (yet) offer things like i18n in templates.
+The documentation is currently only available in english, which is a shame because most of the world does not speak english. If you would like to help making zola available to more people, feel free to translate documentation in your language.
 
-### Pelican notes
+### Programming
+
+Some areas of zola could be improved. If you're just getting started, we maintain a list of [good first issues](https://github.com/getzola/zola/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to get you started hacking on the codebase. If you'd like to tackle some harder problems, you can take a look at the ![ehh] and ![no] features from the comparison table. Additionally, we also have a list of [issues we'd like help on](https://github.com/getzola/zola/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22).
+
+Some (upstream) projects we rely on to build zola could use some help:
+
+- [tera](https://github.com/keats/tera/) rendering engine has some issues marked [`good first issue`](https://github.com/Keats/tera/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) as well as some [`help wanted`](https://github.com/Keats/tera/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+- [pest](https://github.com/pest-parser/pest/issues/489#issuecomment-775114684) parser is [looking for new contributors](https://github.com/pest-parser/pest/issues/489#issuecomment-775114684)
+
+### Packaging
+
+zola is not yet packaged for all systems. An updated list of zola packages can be found [in the documentation](https://www.getzola.org/documentation/getting-started/installation/). Some platforms we would like to have packages for include:
+
+- [Debian](https://debian.org/): zola is currently unavailable, even on `experimental` ; ideally, we would like to support [backports](https://backports.debian.org/) on `stable` as well, because zola is evolving fast
+- [nix](https://nixos.org/): zola on nix is outdated (v0.12.0 as of writing this)
+- [GNU guix](https://guix.gnu.org/): zola is currently unavailable in GNU guix
+
+### Integration
+
+If you are developing free-software user-friendly tooling for content edition (content managing system), or a selfhosting distribution which aims to empower users for self-publication, please get in touch. We'd be glad to help integrate zola with your tools.
```

Comment:
Eeeh I'm a bit iffy on that. It could represent a huge amount of work so I would prefer removing that section.